Homeworks/HW8/fileIO.py

```diff
@@ -0,0 +1,20 @@
+import csv
+
+def read_data(file_name):
+    data = []
+    with open(file_name, newline='') as csvfile:
+        reader = csv.reader(csvfile, delimiter='\t')
+        for row in reader:
+            data.append([row[0], row[1], row[2]])
```

Comment:
```suggestion
        for row1, row2, row1 in reader:
            row = row[0].split(';')
            data.append(row)
```
Если не использовать `split`, то будет ошибка в 9 строке. Не понимаю, почему у вас её не было.